grails-app/views/regions/regions.gsp

```diff
@@ -31,11 +31,11 @@
             </div>
 
             <div class="page-header-links">
-                <a class="page-header-links__link" href="http://ala-test.ut.ee/generic-hub/search#tab-spatial-search">
+                <a class="page-header-links__link" href="${grailsApplication.config.biocache.baseURL}/search#tab-spatial-search">
```

Comment:
Not related to the task at hand but the tab anchors are now without the `tab-` prefix.